tls/s2n_cipher_preferences.c

```diff
@@ -38,8 +38,8 @@ const struct s2n_kem *pq_kems_r1[2] = {
     &s2n_sike_p503_r1,
 };
 
-/* Extension list for round 1 and round 2 PQ KEMs, in order of preference */
-const struct s2n_kem *pq_kems_r1r2[4] = {
+/* Extension list for round 2 and round 1 PQ KEMs, in order of preference */
+const struct s2n_kem *pq_kems_r2r1[4] = {
     &s2n_bike1_l1_r2,
     &s2n_bike1_l1_r1,
```

Comment:
Just noticed this, shouldn't we be preferring `sike_r2` above `bike_r1`?

Reply:
Yeah - good point. Will update.